staging/src/k8s.io/kubectl/pkg/cmd/attach/attach.go

```diff
@@ -157,10 +157,18 @@ type DefaultRemoteAttach struct{}
 
 // Attach executes attach to a running container
 func (*DefaultRemoteAttach) Attach(method string, url *url.URL, config *restclient.Config, stdin io.Reader, stdout, stderr io.Writer, tty bool, terminalSizeQueue remotecommand.TerminalSizeQueue) error {
+	// Legacy SPDY executor is default. If feature gate enabled, fallback
+	// executor attempts websockets first--then SPDY.
 	exec, err := remotecommand.NewSPDYExecutor(config, method, url)
 	if err != nil {
 		return err
 	}
+	if true || cmdutil.RemoteCommandWebsockets.IsEnabled() {
```

Comment:
this will be always executed, no?

Reply:
Yes--just turning on feature flag to test the `FallbackExecutor`. I will remove the `true` before merge.